domain/account/usecases/get-account_test.go

```diff
@@ -0,0 +1,67 @@
+package usecases
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/Erikaa81/banco-erika/domain/entities"
+)
+
+func TestAccount_GetAccount(t *testing.T) {
+	t.Run("should get account id successfuly", func(t *testing.T) {
+		account1, _ := entities.NewAccount("Paulo", "12343456785", "Er5", 10000)
```

Comment:
```suggestion
		expectedAccount, _ := entities.NewAccount("Paulo", "12343456785", "Er5", 10000)
```
como cê não tá mais num array, dá pra ser um pouco mais semântico o nome dessa variável